frontend/src/AudioAnnotator/AudioAnnotator.js

```diff
@@ -353,7 +394,7 @@ class AudioAnnotator extends Component<AudioAnnotatorProps, AudioAnnotatorState>
     const isPresenceMode = !!this.state.task && this.state.task.annotationScope === SCOPE_WHOLE;
 
     const maxId: ?number = this.state.annotations
-      .map(ann => parseInt(ann.id, 10))
+      .map(ann => ann.id, 10)
```

Comment:
I haven't tested this but it looks like a typo, right ? I mean isn't the 10 linked to parseInt and no longer needed ?